hcloud/load_balancers/domain.py

```diff
@@ -354,6 +358,22 @@ def to_payload(self) -> dict[str, Any]:
         return payload
 
 
+class LoadBalancerTargetHealthStatus(BaseDomain):
```

Comment:
This class is not included in the docs. Needs to be added here: https://github.com/hetznercloud/hcloud-python/blob/6bf03cb9ab1203f172e1634d28a99a7cb3210ad0/docs/api.clients.load_balancers.rst?plain=1#L26-L31